<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatorWizard</class>
 <widget class="QWizard" name="CreatorWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreatorWizard</string>
  </property>
  <widget class="QWizardPage" name="racePage">
   <property name="title">
    <string>Race</string>
   </property>
   <property name="subTitle">
    <string>Enter information about your race here</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="raceNameLabel">
      <property name="text">
       <string>Race</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="raceNameEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="racialHPLabel">
      <property name="text">
       <string>Hit Points</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="racialHPSpinBox"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Ability Score Adjustments</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QTableWidget" name="rAbilityAdjWidget">
         <column>
          <property name="text">
           <string>Ability</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Bonus/Penalty</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Racial Abilities</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTableWidget" name="rFeaturesWidget">
         <column>
          <property name="text">
           <string>Feature</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="themePage">
   <property name="title">
    <string>Theme</string>
   </property>
   <layout class="QFormLayout" name="formLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="tNamelineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Ability Score</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="tAbilityComboBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Skill</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="tSkillComboBox"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tFeaturesWidget">
         <property name="columnCount">
          <number>2</number>
         </property>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>6</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>12</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>18</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Feature</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="classPage">
   <property name="title">
    <string>Class</string>
   </property>
   <property name="subTitle">
    <string>Enter your class and level</string>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Class</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="classComboBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Key Ability Score</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_3"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Level</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="levelSpinBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="abilityPage">
   <property name="title">
    <string>Ability Scores</string>
   </property>
   <property name="subTitle">
    <string>Adjust your Ability scores</string>
   </property>
   <layout class="QFormLayout" name="formLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="pointsLabel">
      <property name="text">
       <string>Points available:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="strLabel">
      <property name="text">
       <string>Strength</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="strSpinBox">
      <property name="maximum">
       <number>18</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="dexLabel">
      <property name="text">
       <string>Dexterity</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="dexSpinBox">
      <property name="maximum">
       <number>18</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="conLabel">
      <property name="text">
       <string>Constitution</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="conSpinBox">
      <property name="maximum">
       <number>18</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="intLabel">
      <property name="text">
       <string>Intelligence</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="intSpinBox">
      <property name="maximum">
       <number>18</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="wisLabel">
      <property name="text">
       <string>Wisdom</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="wisSpinBox">
      <property name="maximum">
       <number>18</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="chaLabel">
      <property name="text">
       <string>Charisma</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="chaSpinBox">
      <property name="maximum">
       <number>18</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
